src/main/java/com/ericsson/eiffel/remrem/semantics/schemas/SchemaFile.java

```diff
@@ -24,42 +25,42 @@
  *
  */
 public class SchemaFile {
```

Comment:
You have modified this class to not handle a SchemaFile but all schema files. Hence the name do no longer represent the purpose of this class. Naming of classes and methods is really important! This was not the change I suggested and I do not think it improved the class. I think it should remain a SchemaFile class and perhaps take a few arguments in its constructor to initialize it.